var/spack/repos/builtin/packages/binutils/package.py

```diff
@@ -76,6 +76,8 @@ def install(self, spec, prefix):
 
         if '+libiberty' in spec:
             configure_args.append('--enable-install-libiberty')
+            configure_args.append('--disable-nls')
```

Comment:
Honestly, these changes are completely unrelated to your homepage changes. I would create a separate PR for any binutils changes.